packages/jellyfish-api-core/src/category/mining.ts

```diff
@@ -44,6 +44,18 @@ export class Mining {
   async estimateSmartFee (confirmationTarget: number, estimateMode: EstimateMode = EstimateMode.CONSERVATIVE): Promise<SmartFeeEstimation> {
     return await this.client.call('estimatesmartfee', [confirmationTarget, estimateMode], 'number')
   }
+
+  /**
+   * Attempts to submit new block to network.
+   * See https://en.bitcoin.it/wiki/BIP_0022 for full specification.
+   *
+   * @param {string} hexdata the hex-encoded block data to submit
+   * @param {string} dummy dummy value, for compatibility with BIP22. This value is ignored.
```

Comment:
```suggestion
   * @param {string} dummy value, for compatibility with BIP22. This value is ignored.
```

Reply:
preferably the variable should be part of the statement.

E.g:
- `{string} dummy value for something`
- `{string} hexdata is the hex-encoded`

Otherwise separate by -:

- `{string} dummy - A dummy value for something`